scripts/mktcli.js

```diff
@@ -101,35 +92,34 @@ const main = {
       )
 
     program
-      .command('land-estate <coord>')
-      .description('Get estate info for a LAND (x,y)')
+      .command('asset-data <assetId> <assetType>')
+      .description('Get the data for an asset')
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const contract = eth.getContract('EstateRegistry')
-          const tokenId = await Parcel.encodeTokenId(x, y)
+        asSafeAction(async (assetId, assetType) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
+          const contract = getContractByAssetType(assetType)
 
-          const estateId = await contract.getLandEstateId(tokenId)
+          const data = await contract.tokenMetadata(asset.token_id)
+          const dbData = toDataLog(asset.data)
 
-          log.info(`(land-estate) coords:(${x},${y}) => estateId: ${estateId}`)
+          log.info(`(land-data) coords:(${x},${y})`)
```

Comment:
This could be also an Estate right? maybe we should change it to something more generic